libs/portals/admin/application-system/src/components/StatisticsForm/StatisticsForm.tsx

```diff
@@ -0,0 +1,47 @@
+import { Box, DatePicker } from '@island.is/island-ui/core'
+import { useLocale } from '@island.is/localization'
+import type { ApplicationFilters } from '../../types/filters'
+import { m } from '../../lib/messages'
+
+type Props = {
+  onDateChange: (period: ApplicationFilters['period']) => void
+  dateInterval?: ApplicationFilters['period']
+}
+
+export const StatisticsForm = ({ onDateChange, dateInterval }: Props) => {
+  const { formatMessage } = useLocale()
+
+  return (
+    <Box
+      display="flex"
+      alignItems="center"
+      justifyContent="spaceBetween"
+      flexDirection={['column', 'column', 'column', 'row']}
+    >
+      <Box display="flex" flexDirection={['column', 'column', 'row']}>
+        <Box marginX={[0, 0, 2]} marginY={[2, 2, 0]}>
```

Comment:
Looking at the screenshot, maybe change to `marginRight`, so that the filters line up with rest of content